hw/ip/rom_ctrl/doc/_index.md

```diff
@@ -91,8 +91,8 @@ This is taken by the checker FSM (ignoring ECC bits) and will be compared with t
 
 Once it comes back, the digest is forwarded directly to the [Key Manager]({{< relref "hw/ip/keymgr/doc" >}}).
 It is also compared with the hash that was read from the top eight words of ROM.
-On a match, `check_good_o` is signalled as `On`.
-In either case, `check_done_o` goes high when the calculation is complete.
+On a match, `pwrmgr_data_o.good` is signalled as `On`.
```

Comment:
Is `On` actually an enumerated value anywhere? If not this could maybe just be "high" or `1'b1`

Reply:
Yeah, this is the enum value for `lc_tx_e` in `lc_ctrl_pkg.sv`. Michael suggested I used that as a "multibit bool". This isn't happening in the RTL yet (and we'd need to move the type out of the lifecycle controller if we wanted to do that), but I was leaving it in the documentation as a placeholder ("insert your favourite multi-bit bool here"!). Happy to rip it out if you think that's better?

Reply:
Nah, if it's going to be updated to that at some point then just leave it I reckon